Source/WebKit/Shared/WebPushDaemonConstants.h

```diff
@@ -40,7 +40,7 @@ static constexpr Seconds silentPushTimeoutForProduction { 30_s };
 static constexpr Seconds silentPushTimeoutForTesting { 1_s };
 
 constexpr auto protocolVersionKey = "protocol version"_s;
-constexpr uint64_t protocolVersionValue = 4;
+constexpr uint64_t protocolVersionValue = 5;
```

Comment:
Do we need to upgrade version for this change? The message format is the not changed. Is it because the old code does not how to handle failure?

Reply:
I added a new IPC to `PushClientConnection.messages.in` and I think technically we ought to be incrementing this version any time we add an IPC message or change the order of IPC messages in this file.

I was actually thinking of also embedding the WebKit version in the IPC message (like we do for AuxiliaryProcess startup), but I guess maybe we can do that in a separate patch.